src/deprecations.jl

```diff
@@ -152,3 +152,4 @@ Base.@deprecate_binding in_linear_system is_in_linear_system
 # deprecated for 1.3
 @deprecate acting_domain(C::GroupCoset) acting_group(C)
 @deprecate acting_domain(Omega::GSet) acting_group(Omega)
+@deprecate grid_morphism lattice_homomorphism
```

Comment:
Why deprecate `lattice_homomorphism`?

Unless I am mistaken, we wish to replace `grid_morphism` by `lattice_homomorphism`. So we deprecate `grid_morphism` and newly export `lattice_homomorphism`.

(Minor nitpick: I remember there is a syntax for deprecations, so that users see a message like "The method xyz is no longer supported. Please use ... instead.")

Reply:
Nvm... This is what this already does. ;)

Reply:
This syntax here deprecates grid_morphism in favor of lattice_homomorphism.
And this macro enables exactly that message once a user uses the deprecated function with --depwarn=yes enabled